.circleci/config.yml

```diff
@@ -2,7 +2,7 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@2.10.0
+  shared: getoutreach/shared@dev:local
```

Comment:
I think you'll need to revert this line, it's a nuance of devbase replacing itself. Will need to add some logic to stencil-circleci to use some other version for this. Dunno yet.